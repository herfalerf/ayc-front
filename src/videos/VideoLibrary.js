import React from "react";
import VideoList from "./VideoList";

// Video library
//
// Displays video list component
//
// Routed at /resources
//
// Routes -> VideoLibrary

function VideoLibrary() {
  console.debug("VideoLibrary");

  return (
    <div>
      <h3>Get Started with Our Resource Library</h3>
      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
      <VideoList usename="true" />
    </div>
  );
}

export default VideoLibrary;
