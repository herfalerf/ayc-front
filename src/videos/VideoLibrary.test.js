import React from "react";
import { render } from "@testing-library/react";
import VideoLibrary from "./VideoLibrary";

it("renders without crashing", function () {
  render(<VideoLibrary />);
});

it("matches snapshot", function () {
  const { asFragment } = render(<VideoLibrary />);
  expect(asFragment()).toMatchSnapshot();
});
