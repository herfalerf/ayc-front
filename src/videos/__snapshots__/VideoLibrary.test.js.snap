// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="MuiGrid-root makeStyles-upper-13 MuiGrid-container MuiGrid-align-items-xs-flex-start MuiGrid-justify-content-xs-center"
    >
      <div
        class="MuiGrid-root makeStyles-headerItem-14 MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <h2
          class="MuiTypography-root makeStyles-subItem-16 MuiTypography-h2"
        >
          <span
            class="makeStyles-textUnderline-17"
          >
            Get Started
          </span>
           with our Resource Library
        </h2>
        <p
          class="MuiTypography-root makeStyles-subItem-16 MuiTypography-subtitle1"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit.
        </p>
      </div>
      <div
        class="MuiGrid-root makeStyles-headerVideo-15 MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4"
      >
        <div
          class="LoadingSpinner"
        >
          Loading ...
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root makeStyles-videoBox-20 MuiGrid-container"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-content-xs-center MuiGrid-grid-xs-12"
      >
        <p
          class="MuiTypography-root MuiTypography-body1"
        >
          <span
            aria-hidden="true"
            class="material-icons MuiIcon-root makeStyles-logoMark-18"
          >
            <img
              alt="logo mark"
              class="makeStyles-logoMark-18"
              src="ic_logomark.svg"
            />
          </span>
          <span
            class="MuiTypography-root makeStyles-logoText-19 MuiTypography-subtitle1"
          >
            Videos
          </span>
        </p>
      </div>
      <div
        class="LoadingSpinner"
      >
        Loading ...
      </div>
    </div>
  </div>
</DocumentFragment>
`;
